main.go

```diff
@@ -1,9 +1,26 @@
 package main
 
 import (
+	"flag"
 	"fmt"
+	"github.com/LinuxAtApp/matterleast/servercom"
 )
-
+/*
+Main usage: `go run main.go -u <username> -p <password> <url>
+Package demontrates simple login functions using the servercom package's methods.
+*/
 func main() {
 	fmt.Println("matterleast")
-}
+	username := flag.String("u", "", "Username")
+	password := flag.String("p", "", "Password")
+	flag.Parse()
+	url := flag.Arg(0)
+	//Creates client and logs user in.
+	client:= *servercom.Startup(url, *username, *password)
```

Comment:
@aThorp96 Why are you de-referencing the pointer that `servercom.Startup()` returns?